testcases.py

```diff
@@ -266,6 +266,38 @@ def check(self):
         return True
 
 
+class TestCaseLongRTT(TestCaseHandshake):
+    @staticmethod
+    def abbreviation():
+        return "Y"
+
+    @staticmethod
+    def name():
+        return "longrtt"
+
+    @staticmethod
+    def testname(p: Perspective):
+        return "handshake"
+
+    @staticmethod
+    def scenario() -> str:
+        """ Scenario for the ns3 simulator """
+        return "simple-p2p --delay=750ms --bandwidth=10Mbps --queue=25"
+
+    def check(self):
+        if not super(TestCaseLongRTT, self).check():
+            return False
+        num_ch = 0
+        for p in self._client_trace().get_initial(Direction.FROM_CLIENT):
+            if hasattr(p, "tls_handshake_type"):
+                if p.tls_handshake_type == "1":
+                    num_ch += 1
+        if num_ch < 2:
+            logging.info("Expected at least 2 ClientHellos. Got: %d", num_ch)
+            return False
```

Comment:
This assumes a handshake timeout of < 750ms. Of course, this is reasonable, but is there a reason to fail an implementation for not doing this? This isn't a performance test. I would suggest not doing anything in `check()`; the pass/fail status of the test should be enough for this purpose.

Reply:
This assumes a handshake timeout of > 1500ms, which, as far as I can see, is given for all implementations currently on the interop runner. So what this check does is just to make sure that the ClientHello was retransmitted.

Reply:
Right, sorry, 1500ms. That makes it more reasonable. In practice, you are right that implementations will retransmit well before this time, but I was arguing that the checker here is checking for expected behavior, not required behavior, which makes it theoretically more fragile. That said, I think this is a large enough RTT that it's really a pedantic discussion at this point.